tests/src/test/scala-2.11+/cats/tests/RegressionTests.scala

```diff
@@ -1,25 +0,0 @@
-package cats
-package tests
-
-
-import ExtraRegressionSuite._
-
-class ExtraRegressionSuite extends CatsSuite {
```

Comment:
Why are we deleting this? 🤔

Reply:
oops I did the renamed but didn't include the renamed file in the git. just pushed. good catch.